src/GAP.jl

```diff
@@ -182,7 +183,7 @@ function initialize(argv::Array{String,1})
     # detect by checking for the value of __JULIAINTERNAL_LOADED_FROM_JULIA).
     append!(argv, ["-c", """BindGlobal("__JULIAINTERNAL_LOADED_FROM_JULIA", true );"""])
 
-    ccall(
+    @sync ccall(
```

Comment:
Perhaps it would be better wrap the whole content of this function into a single `@sync` block -- or even simply the call to `initialize`?

Reply:
Sounds good.